queries.sql

```diff
@@ -0,0 +1,9 @@
+SET SQL_SAFE_UPDATES = 0;
+
+UPDATE teams 
+    SET teamAbbreviation = 'WAS' 
+    WHERE id = (SELECT id WHERE teamAbbreviation = 'WSH');
+    
+SELECT startTime, homeScore, awayScore FROM games WHERE startTime <= '2019-06-06 17:00:00';
```

Comment:
The question is _on_ June 6th _before_ 5pm. This query will give you everything _before_ June 6 at 5pm. So you'd get games played on June 5th, etc. Also there is an interesting English problem here. Does `before 5pm` include 5pm? Maybe, you'd have to ask the person who needs the data.

`SELECT startTime, homeScore, awayScore FROM games WHERE startTime >= '2019-06-06 00:00:00' AND startTime < '2019-06-06 17:00:00';`